src/app/app-routing.module.ts

```diff
@@ -6,9 +6,11 @@ import {PageNotFoundComponent} from "./user/page-not-found/page-not-found.compon
 import {WelcomeComponent} from "./user/welcome/welcome.component";
 import {AuthGuard} from "./guard/auth.guard";
 import {DetailContainerComponent} from "./hero/detail/detail-container/detail-container.component";
+import {LoginComponent} from "./user/login/login.component";
 
 const routes: Routes = [
   {path: "welcome", component: WelcomeComponent},
+  {path: "login", component: LoginComponent, outlet: "loginPage"},
```

Comment:
can you rename it to modal or popup so that this outlet can be used by all components that could be / would be modals